infra/modules/database/role_manager/manage.py

```diff
@@ -89,8 +89,11 @@ def configure_database(conn: Connection) -> None:
     schema_name = os.environ.get("DB_SCHEMA")
     database_name = os.environ.get("DB_NAME")
 
+    # This is redundant as of Postgres 15, but leaving in for
+    # compatibility with projects using older versions of Postgres.
```

Comment:
optional nit: can you explain why it's redundant e.g.
"Starting with PostgreSQL 15, the CREATE privilege on public schema is already revoked/removed from all users except the database owner. However, we are still revoking explicitly for projects using older versions of Postgres"